apps/dashboard/src/app/knowledge-box/knowledge-box-process/knowledge-box-processes.component.html

```diff
@@ -11,13 +11,24 @@ <h2>{{ 'stash.settings.processes.' + type | translate }}</h2>
     <div [class.running]="running[type]">
       <p>{{ 'stash.settings.processes.' + type + '.info' | translate }}</p>
 
-      <pa-select
-        [label]="'stash.settings.processes.training.select-labelsets' | translate"
-        [(value)]="selectedLabelsets[type]"
-      >
-        <pa-option value="">{{ 'generic.all' | translate }}</pa-option>
-        <pa-option *ngFor="let option of labelsets | async" [value]="option.value">{{ option.title }}</pa-option>
-      </pa-select>
+      <div class="labelset-selection">
+        <pa-button class="labelset-selection-button"
+                   [paPopup]="dropdown">
+          {{ 'stash.settings.processes.training.select-labelsets' | translate }}
+          <pa-icon [name]="open[type] ? 'chevron-up' : 'chevron-down'"></pa-icon>
+        </pa-button>
+        <div class="selection">{{ selectedLabelsets[type].join(', ') }}</div>
+      </div>
+
+      <pa-dropdown #dropdown
+                   (onOpen)="open[type] = true"
+                   (onClose)="open[type] = false">
+        <pa-option *ngFor="let labelset of labelsets | async"
+                   dontCloseOnSelect>
+        <pa-checkbox [value]="selectedLabelsets[type].includes(labelset.value)"
```

Comment:
in the design you showed me, there was an option to select all the labelsets. Is this something we don't want at the end?

Reply:
oops, I forgot. Its added now.